pokedex/model/template/csr/write/fcsr_003.asl

```diff
@@ -1,4 +1,15 @@
-FFLAGS = value[4:0];
-FRM = value[7:5];
+var mr = ReadCSR(MSTATUS_IDX);
+assert mr.is_ok;
+mr.value[MSTATUS_FS_HI:MSTATUS_FS_LO] = '11'; // set dirty
+let mw = WriteCSR(MSTATUS_IDX, mr.value);
+assert mw.is_ok;
```

Comment:
This could be abstracted into a function, e.g. makeFsDirty. Every F instructions will contain such logics.